pyuvdata/uvdata/tests/test_ms.py

```diff
@@ -19,17 +19,37 @@
 
 
 @pytest.fixture(scope="session")
-def nrao_uv():
+def nrao_uv_main():
+    # This file is known to be made with CASA's importuvfits task, but apparently the
+    # history table is missing, so we can't infer that from the history. This means that
+    # the uvws are not flipped/data is not conjugated as they should be. Fix that.
+
     uvobj = UVData()
     testfile = os.path.join(DATA_PATH, "day2_TDEM0003_10s_norx_1src_1spw.ms")
     uvobj.read(testfile)
+    # uvobj.uvw_array *= -1
+    # uvobj.data_array = np.conj(uvobj.data_array)
```

Comment:
Oh, wait. If these lines are out then should we take out the earlier comment?

Reply:
Deleted.